_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+## Terminal: Support for changing the font size
+
+Users can now change the font size in the emulator window on the
+terminal page.
+
+![New font size selector in the terminal](/images/terminal-font-size.png)
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+disks.  This was previously only possible when creating new disks.  In
+order to edit the cache mode, any machines using the volume need to be
+powered off.
+
+![Cache mode editing for machines](/images/edit-vm-cache-mode.png)
+
+### Logs: Link to related services page
+
+When viewing the details of a log entry, there's now a link directly to
+the relevant service page, if applicable.
+
+![Log entry linking to service](/images/log-link-to-service.png)
+
+### SELinux: Restyle to resemble other pages
+
+The SELinux page has been restyled to better match the style of other
+pages in Cockpit.
+
+![SELinux page with the new style](/images/selinux-restyled.png)
+
+### Packaging: Removed ./configure options for distribution specific packages
+
+The `--with-vdo-package`, `--with-nfs-client-package`,
+`--with-appstream-config-packages`, and `--with-appstream-data-packages`
+options for `./configure` got removed. This information is now contained
+in the respective manifest files, for example for the [Storage
+page](https://github.com/cockpit-project/cockpit/blob/master/pkg/storaged/manifest.json.in).
+This makes the manifest files portable across distributions. They can
+still be adjusted by administrators through [override.json
+files](https://cockpit-project.org/guide/latest/packages.html#package-replace).
```

Comment:
```suggestion
Several distribution-specific options for `./configure` were removed, including:

- `--with-vdo-package`
- `--with-nfs-client-package`
- `--with-appstream-config-packages`
- `--with-appstream-data-packages`

This information is now contained in respective manifest files (`storaged` and `apps`), making manifest files more portable across distributions.

These values can still be adjusted by administrators in `override.json` files.
```

Reply:
I'm not sure about the "(`storaged` and `apps`)" part; please someone correct this if wrong. I do see the vdo/nfs align to Storage and Appstream would make sense for Applications.